lib/mcinstall/index.js

```diff
@@ -0,0 +1,61 @@
+import {BaseServicePlist} from '../base-service';
+import {fs, plist} from '@appium/support';
+
+
+const MC_INSTALL_SERVICE_NAME = 'com.apple.mobile.MCInstall';
+const ACKNOWLEDGED = 'Acknowledged';
+
+function checkACK (res, name) {
+  if (res.Status !== ACKNOWLEDGED) {
+    throw new Error(`${name} error: ${JSON.stringify(res)}`);
+  }
+  return res;
+}
+
+class MCInstallProxyService extends BaseServicePlist {
+  constructor (socketClient) {
+    super(socketClient);
+  }
+
+  /**
+   * Get all profiles of iOS devices
+   * @returns {Promise<Object>}
+   */
+  async getProfileList () {
+    const res = await this._plistService.sendPlistAndReceive({RequestType: 'GetProfileList'});
+    return checkACK(res, 'getProfileList');
+  }
+
+  /**
+   * Install profile to iOS device
+   * @param {String} path  path of the profile file e.g: /Downloads/charles-certificate.pem
```

Comment:
it would probably make sense to explicitly mention which certificate file formats are acceptable.

Reply:
I only know that PEM and CER formats can be installed, there may be many more